src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the checklist, the `get()` method on the service layer should return the entity directly, but on the dao layer, it should return `Optional`. Please adjust the return type here to `MovieSession` and handle the `Optional` in the service implementation.